homeassistant/components/flipr/entity.py

```diff
@@ -5,7 +5,7 @@
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
 from .const import ATTRIBUTION, DOMAIN, MANUFACTURER
-from .coordinator import FliprDataUpdateCoordinator
+from .coordinator import BaseDataUpdateCoordinator
 
 
 class FliprEntity(CoordinatorEntity):
```

Comment:
We also want to type the coordinator in the base entity.

```suggestion
class FliprEntity(CoordinatorEntity[...Coordinator]):
```